--- ../arch/mips/jz4770/clock.c	2021-02-20 10:18:19.000000000 +0100
+++ ../arch/mips/jz4770/clock_mod.c	2021-02-28 08:57:30.247390486 +0100
@@ -193,7 +193,7 @@
 }
 
 #define FVCO_MIN  300*1000*1000	/* 300 MHz */
-#define FVCO_MAX  1080*1000*1000	/* 1,1 GHz */
+#define FVCO_MAX  1000*1000*1000	/* 1 GHz */
 
 /* The frequency after the input divider must be between 10 and 50 MHz.
 The highest divider yields the best resolution. */
