--- ../drivers/gpu/vivante/v4/gc_hal_kernel_driver.c	2021-02-20 10:18:19.000000000 +0100
+++ ../drivers/gpu/vivante/v4/gc_hal_kernel_driver_mod.c	2021-02-28 09:02:45.836698655 +0100
@@ -695,18 +695,19 @@
 {
 #ifdef CONFIG_MACH_JZ4770
     {
-        /* JZ4770 GPU CLK2x 360MHz -- 540MHz --this is the correct value for 1080Mhz overclock and 360Mhz downclock (in userspace the min value is 360Mhz), because the gpu clock is cpuclock/2 */
-#define GPU_CLK_MAX 540000000
+        /* JZ4770 GPU CLK2x 100MHz -- 500MHz */
+#define GPU_CLK_MAX 500000000
         unsigned int GPUCDR_VAL=0;
         int div;
         int gpu_use_pll1 = 1;
         unsigned int pll_clk;
         unsigned int gpu_clk = 0;
 
-		/*gpu is pll0
+        /* Right now: hardcode PLL0.
+	 * Later: use generic clock interface.
         pll_clk = cpm_get_pllout1();
         if ( pll_clk == 0 )*/ {
-            gpu_use_pll1 = 0;
+            gpu_use_pll1 = 0;   /* use pll0 */
             pll_clk = cpm_get_pllout();
             if ((INREG32(CPM_CPCCR) & CPCCR_PCS) != 0 )
             pll_clk /= 2;
@@ -714,7 +715,7 @@
 
         for ( div=1; div <= ((GPUCDR_GPUDIV_MASK>>GPUCDR_GPUDIV_LSB)+1); div++ ) {
             gpu_clk = pll_clk/div;
-            if ( gpu_clk < GPU_CLK_MAX+1 )
+            if ( gpu_clk < GPU_CLK_MAX )
                 break;
         }
 
